src/ui/Facet/FacetSearchDropdownNavigation/DefaultSearchDropdownNavigator.ts

```diff
@@ -0,0 +1,73 @@
+import { first, last } from 'underscore';
+import { $$, Dom } from '../../../utils/Dom';
+
+export interface ISearchDropdownNavigator {
+  nextFocusableElement: () => void;
```

Comment:
Humm, name is a bit misleading perhaps ? 
It would suggest that it returns or `get` the next focusable element, instead of actually doing the action of moving the focus in the input

Suggestion: 
```
focusNextElement()
previousPreviousElement()
```